src/engine/server/sv_client.cpp

```diff
@@ -528,7 +528,7 @@ the same as cl->downloadClientBlock
 */
 void SV_NextDownload_f( client_t *cl, const Cmd::Args& args )
 {
-	int block;
```

Comment:
What's wrong here? It's initialized by `Str::ParseInt`.

If compilers are going to give bogus 'maybe unitialized' warnings, it's better to turn off those warnings rather than adding nonsensical assignments everywhere.

Reply:
oh, right, value is guaranteed to be written into